public/templates/index.dust

```diff
@@ -18,7 +18,9 @@
               <img src="/img/logo.png">
               <p class="lead">{@pre type="content" key="subtitle"/}</p>
   <div class="input-group">
-      <input id="ignoreSearch" type="text" placeholder="{@pre type="content" key="searchPlaceholder"/}" tabindex="1" class="input form-control">
+
+      <input id="ignoreSearch" type="text"  tabindex="1" class="input form-control">
```

Comment:
Nitpick alert: looks like there's 2 spaces before `tabindex` where there should probably be one. Also, are the new lines necessary?